tasks/libs/civisibility.py

```diff
@@ -74,7 +74,9 @@ def get_test_link_to_test_on_main(suite_name: str, test_name: str):
     }
     query_string = to_query_string(query_params)
     quoted_query_string = quote(string=query_string, safe="")
-    return f"{TEST_VISIBILITY_URL}?query={quoted_query_string}"
+    full_url = f"{TEST_VISIBILITY_URL}?query={quoted_query_string}"
+    url_element = f'<a href="{full_url}" class="gl-reset-color! gl-text-decoration-underline" rel="nofollow noopener noreferrer">Test Visibility</a>'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏

You could create a utility function to make it easier to create this kind of link, such as: 

```python
def create_link(url, text):
	return f'<a href="{url}" class="gl-reset-color! gl-text-decoration-underline" rel="nofollow noopener noreferrer">{text}}</a>'
```

Reply:
Indeed, as it was a single call I didn't "invest" in doing a method. Will for sure deserve it if we need to create more elements like this